### PR TITLE
Fixed typos and grammatical errors

### PR DESCRIPTION
**Pull request by jcomack**
Added some missing words, fixed some typos and fixed some grammatical errors.

### CODE REVIEW

File: en/core-libraries/logging.rst [hunk 1]
```diff
@@ -24,11 +24,11 @@ Creating and configuring log streams
 
 Log stream handlers can be part of your application, or part of
 plugins. If for example you had a database logger called
-``DatabaseLog``. As part of your application it would be placed
-in ``app/Lib/Log/Engine/DatabaseLog.php``. As part of a plugin it
+``DatabaseLog`` as part of your application, it would be placed
+in ``app/Lib/Log/Engine/DatabaseLog.php``. If you had a database logger as part of a plugin, it
```
Comment (ravage84) [inline en/core-libraries/logging.rst]: New line after "database", please

File: en/core-libraries/logging.rst [hunk 2]
```diff
@@ -24,11 +24,11 @@ Creating and configuring log streams
...
 would be placed in
 ``app/Plugin/LoggingPack/Lib/Log/Engine/DatabaseLog.php``. When
-configured ``CakeLog`` will attempt to load Configuring log streams
+configured, ``CakeLog`` will attempt to load Configuring log streams, which
```
Comment (ravage84) [inline en/core-libraries/logging.rst]: Why is "Configuring"  with a capital C?

**Comment by ravage84** [comment]
Looks good apart from my comments. Thanks again.

**Comment by jcomack** [comment]
Didn't notice that nor did I feel it needed to be addressed :)
Anyways, you're welcome.

**Comment by markstory** [comment]
I'll fix up the formatting nitpicks post merge.

**Pull request merged**